hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/AbstractKeyDeletingService.java

```diff
@@ -456,6 +460,7 @@ public long optimizeDirDeletesAndSubmitRequest(long remainNum,
               " totalRunCount: {}",
           dirNum, subdirDelNum, subFileNum, (subDirNum - subdirDelNum),
           Time.monotonicNow() - startTime, rnCnt);
+      metrics.incrementDirectoryDeletionTotalMetrics(dirNum + subdirDelNum, subdirMoved, subFileNum);
```

Comment:
validate if {deletedDirsCount, movedDirsCount, movedFilesCount} as variable, PurgeDir metrics {NumSubDirectoriesMoved,  NumSubFilesMoved, NumDirPurged} and Metrics here {dirDel, dirMove, fileMove} are same value, else this will create confusion between metrics, and also with logs.